frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -31,9 +33,20 @@ const PipelinesOverview: React.FC<PipelinesOverviewProps> = ({
   const {
     metadata: { name, namespace },
   } = pipeline;
+  const [showAlert, setShowAlert] = React.useState(isPipelineNotStarted(name, namespace));
```

Comment:
This is not reactive 😞  

I'll need to do some investigation into why we can't just use it in the render.

Reply:
Ah okay, I see what is happening here... yeah your original solution made more sense. I'm not sure if we can entirely base our data on the sessionStorage 🤔  (which was my original reason for my suggestion)

Reply:
Yeah, I think we can't get a "single state" of storage here... which is unfortunately. Okay, reset it back to the useEffect hook and we should be good to go.

```
  const [showAlert, setShowAlert] = React.useState(isPipelineNotStarted(name, namespace));

  React.useEffect(() => {
    setShowAlert(isPipelineNotStarted(name, namespace));
  }, [name, namespace]);
```

Sorry for this round-about solution... you had it right the first time.